client/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:10
 
-WORKDIR /usr/src/app
+WORKDIR /app
```

Comment:
I'm not married to `/usr/src/app` but what wasn't working with it? If it needs to be changed the volume mapping in docker-compose.yml should also be updated.

Reply:
Admittedly this was a bad suggestion.  The actual problem has something to do with mounting the volumes in the docker-compose.yml file.  For some reason 'nodemon' does not get installed if the WORKDIR in the dockerfile is the same as the volume path in the docker-compose file.   This of course causes the "start" script to fail and the container shuts itself down.  Changing this path seems to fix the issue, but at this time I'm not certain why.  I can close this pull request until I understand what's actually going on.

Reply:
nodemon is only a devDependency (and the only one), it's possible this is an npm issue. The current setup seems to be working for me with the `node:10` image but that gives us a one out of two track record. What Docker version and OS are you on?

Reply:
Docker version 18.06.0-ce, build 0ffa825 running on High Sierra

Reply:
Weird, same here on both counts. Let's do a Skype/Zoom/whatever and see what's up.